lib/core/utils/dq-element.js

```diff
@@ -82,7 +86,9 @@ function DqElement(elm, options = null, spec = {}) {
   if (!axe._audit.noHtml) {
     this.source = this.spec.source ?? getSource(this._element);
   }
-}
+
+  return this;
```

Comment:
Memoize requires something to be returned otherwise it caches the function as `undefined`